src/components/RecycleBinScreen.js

```diff
@@ -60,6 +62,9 @@ const RecycleBinScreen = () => {
 
   return (
     <Main>
+      <Typography variant="h4" align="left" id={TITLE_TEXT_ID}>
+        {t('Build and share rich educational resources')}
```

Comment:
It's the Recycle Bin screen, so perhaps we want a more adapted text?

Reply:
![image](https://user-images.githubusercontent.com/44131581/182572584-b8b12fe8-152c-471c-a39e-13abf5ca9b1c.png)
![image](https://user-images.githubusercontent.com/44131581/182572641-0138ac95-bc57-4a71-ad90-79231c522da1.png)
![image](https://user-images.githubusercontent.com/44131581/182572686-5a17e24f-5ebb-4bc8-b130-7f2c369b4deb.png)
![image](https://user-images.githubusercontent.com/44131581/182572726-e7c253ee-fc43-4fde-8c97-446ad7bd702c.png)
Yeah, maybe we could have adaptive texts for each page.